src/config/navigation/defaultStackConfig.tsx

```diff
@@ -139,10 +139,19 @@ export const defaultStackConfig = ({
     {
       routeName: ScreenName.EncounterUserDetail,
       screenComponent: EncounterUserDetailScreen,
-      screenOptions: ({ navigation }) => ({
+      screenOptions: ({ navigation, route }) => ({
         title: texts.screenTitles.encounterHome,
         headerLeft: () => (
-          <HeaderLeft onPress={() => navigation.navigate(ScreenName.EncounterHome)} />
+          <HeaderLeft
+            onPress={() => {
+              // @ts-expect-error we are lacking proper param types here
+              if (route.params?.fromPoll) {
```

Comment:
starting reviewing with this line makes me think "what :o" ... `fromPoll`? and i start reading the commit message again without finding anything about it.

Reply:
with coming to `navigation.push(ScreenName.EncounterUserDetail, { data, fromPoll: true });` later, this makes more sense.

could have been described, that we need this way of "communication" between the components